hil/ext/auth/database.py

```diff
@@ -60,6 +61,18 @@ def set_password(self, password):
                          db.Column('project_id', db.ForeignKey('project.id')))
 
 
+@rest_call('GET', '/auth/basic/user', schema=Schema({}))
+def list_users():
+    """List all users"""
+    get_auth_backend().require_admin()
+    users = db.session.query.all()
```

Comment:
This wouldn't work, you haven't specified what table to query.

Reply:
i shoud have: `User.query.all()` Thanks!